packages/appcache/appcache-server.js

```diff
@@ -39,6 +39,13 @@ var browserDisabled = function (request) {
   return disabledBrowsers[request.browser.name];
 };
 
+function isDynamic(resource) {
+  return resource.type === 'dynamic js' ||
+    (resource.type === 'json' &&
+      resource.url.startsWith('/dynamic/') &&
```

Comment:
This URL prefix may change after #9439, which makes these URLs look like `/__browser/dynamic/...` but that's OK—this is mostly a note to myself to update this code in that PR.